crates/precompile-evm-to-native-swap/EvmToNativeSwap.sol

```diff
@@ -19,4 +19,14 @@ interface CurrencySwap {
    * @return success Whether or not the swap was successful.
    */
   function swap(bytes32 nativeAddress) external payable returns (bool success);
+
+  /**
+   * Event emitted when a transfer has been performed.
+   * Selector: 69d31d1d87c1206beee49fbab11570a7f001121cf21fbb234d5b0a2473fa5c58
+   *
+   * @param from The EVM account id the tokens withdrawed from.
+   * @param to The Native account id the tokens deposited to.
+   * @param value The amount of tokens swapped.
+   */
+  event Swap(address indexed from, bytes32 indexed to, uint256 value);
```

Comment:
To do; check if the name `Swap` in meaningful here

Reply:
Didn't get. It's checked here - https://github.com/humanode-network/humanode/blob/826c5b1703c61507a8efc206677e021627ab5926/utils/e2e-tests/ts/tests/swap/evmToNative.ts#L88

Reply:
I've discovered that we missed it when decided to walk through `precompile-native-currency` to recheck precompile code structure.

Reply:
I mean, I left this note for myself to remind to check if having it called `Swap` vs say `Swapped` makes sense